reference/v8.json

```diff
@@ -790,15 +796,24 @@
       "property-function": true,
       "values": [
         "map",
-        "viewport"
+        "viewport",
+        "auto"
       ],
-      "doc": "Aligns text to the plane of the `viewport` or the `map` when the map is pitched. Matches `text-rotation-alignment` if unspecified.",
+      "default": "auto",
+      "doc": "Aligns text to the plane of the `viewport` or the `map` when the map is pitched. Matches `text-rotation-alignment` if `auto`.",
```

Comment:
While you're here, I think the use of backticks here would contribute to confusing documentation in the iOS and macOS SDKs. It'll say something like "Aligns text to the plane of the `MGLTextPitchAlignmentViewport` or the `MGLTextPitchAlignmentMap` when the map is pitched." But the intent is to say "plane of the viewport when set to `viewport`".